auth-web/src/stores/business.ts

```diff
@@ -343,6 +343,26 @@ export const useBusinessStore = defineStore('business', () => {
         addBusinessTypeforSP(filingBody, business)
         break
       }
+
+      case FilingTypes.CONTINUATION_IN: {
+        filingBody = {
+          filing: {
+            business: {
+              legalType: business.nameRequest.legalType
+            },
+            header: {
+              accountId: currentOrganization.value.id,
+              name: filingType
+            },
+            continuationIn: {
+              nameRequest: {
+                legalType: business.nameRequest.legalType,
+                nrNumber: business.nameRequest.nrNumber
```

Comment:
Shouldn't this be `business.businessIdentifier || business.nameRequest.nrNumber`?

Reply:
This looks right to me.

The business identifier is never a NR number, right? Or are you thinking that the business number might be the description? In that case it would go in the nameRequest.legalName field -- but leave that for a separate ticket to worry about. Here, you're just creating the initial draft.

Reply:
OK, sounds good Sev. We'll leave it to a separate ticket to flesh that out later. Resolving this conversation. Thanks!